src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let messege = '';
+
+  if (!person.isMarried) {
+    messege = `I am not married`;
+  } else {
+    messege = `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`;
+  }
```

Comment:
you can just create one variable and use condition at jsx
```suggestion
  const partnerStatus = sex === 'm' ? 'wife' : 'husband';
```